src/lib.rs

```diff
@@ -1,20 +1,37 @@
 #![cfg_attr(not(test), no_std)]
+
 //! Create a trusted carrier with a new lifetime that is guaranteed to be
-//! unique. When you call [`make_guard!`] to make a unique lifetime, the macro
-//! creates a [`Guard`] to hold it. This guard can be converted `into` an
-//! [`Id`], which can be stored in structures to uniquely "brand" them. A different
-//! invocation of the macro will produce a new lifetime that cannot be unified.
-//! These types have no safe way to construct them other than via
-//! [`make_guard!`].
+//! unique among other trusted carriers. When you call [`make_guard!`] to make a
+//! unique lifetime, the macro creates a [`Guard`] to hold it. This guard can be
+//! converted `into` an [`Id`], which can be stored in structures to uniquely
+//! "brand" them. A different invocation of the macro will produce a new
+//! lifetime that cannot be unified. These types have no safe way to construct
+//! them other than via [`make_guard!`] or `unsafe` code.
```

Comment:
nitpick: unsafe code is not a "safe way" to construct something. So maybe we just say:

```
The only way to construct these types is with [`make_guard!`] or `unsafe` code.
```